programs/voter-stake-registry/Cargo.toml

```diff
@@ -19,8 +19,8 @@ test-bpf = []
 
 [dependencies]
 # a) for deployment use these
-anchor-lang = "0.20.1"
-anchor-spl = { version = "0.20.1" }
+anchor-lang = { version = "0.24.2", features = ["init-if-needed"] }
```

Comment:
nitpicks:
1/ anchor-lang 0.24.2 depends on `solana-program = "~1.9.13"`, should we just bump solana dependencies for the sake of completeness?
2/ did you update your local anchor-cli installation to this version also?

Reply:
1/ yes let's
2/ yes, always use the matching anchor-cli version (I just don't have an `anchor` binary anymore, just `anchor-v0.24.2`)

Reply:
(note that updating the solana dependency isn't stricly necessary since both the vsr dep and anchor dep are >= dependencies)